### PR TITLE
riotbuild: bump riscv toolchain to 10.2

### PR DESCRIPTION
**Pull request by aabadie**
The minor fixes correspond to this warning raised by Docker: `apt does not have a stable CLI interface. Use with caution in scripts.`. So use apt-get instead which is consistent with the rest of the commands.

There's a small name duplication removed with coccinelle package using an ARG.

### CODE REVIEW

**Comment by aabadie** [comment]
bors try

File: riotbuild/Dockerfile [hunk 1]
```diff
@@ -151,7 +151,7 @@ ENV MIPS_ELF_ROOT /opt/mips-mti-elf/${MIPS_VERSION}
 ENV PATH ${PATH}:${MIPS_ELF_ROOT}/bin
 
 # Install RISC-V binary toolchain
-ARG RISCV_VERSION=10.1.0-1.1
+ARG RISCV_VERSION=10.2.0-1.2
```
Comment (fjmolinas) [inline riotbuild/Dockerfile:154]: Unrelated?

**Review by fjmolinas — COMMENTED** [review]
I don't think the RISCV bump belongs here, can you split that one out? Otherwise all good.

File: riotbuild/Dockerfile (hunk 1 above)
Comment (aabadie) [reply riotbuild/Dockerfile:154]: That's the goal of this PR :)

Comment (fjmolinas) [reply riotbuild/Dockerfile:154]: Ahh sorry, I didn't read the PR title, only the description.

**Comment by fjmolinas** [comment]
@aabadie I wonder if we should wait for the release before getting this in, although minor version bump issues could still surface...

**Comment by kaspar030** [comment]
@aabadie would you please split out the minor fixes? Those we could merge anytime (I'd ACK them right away).

**Comment by aabadie** [comment]
> would you please split out the minor fixes?

Done in #175

**Comment by fjmolinas** [comment]
Lets get this in after https://github.com/RIOT-OS/riotdocker/pull/180

**Comment by fjmolinas** [comment]
bors merge

**Review by fjmolinas — APPROVED** [review]
ACK

**Comment by fjmolinas** [comment]
bors merge
